ChangeLog.d/clean_pem_buffers.txt

```diff
@@ -0,0 +1,13 @@
+Bugfix
+  * As an optimisation to stop large buffers being written to the stack the
```

Comment:
This would be the second-longest changelog entry ever. Surely it can be summarized/